core/src/main/java/org/apache/accumulo/core/clientImpl/ConditionalWriterImpl.java

```diff
@@ -372,11 +372,8 @@ private void reschedule(SendTask task) {
 
       for (int i = 1; i < mutations.size(); i++) {
         for (Entry<KeyExtent,List<QCMutation>> entry : mutations.get(i).getMutations().entrySet()) {
-          List<QCMutation> list = tsm.getMutations().get(entry.getKey());
-          if (list == null) {
-            list = new ArrayList<>();
-            tsm.getMutations().put(entry.getKey(), list);
-          }
+          List<QCMutation> list = tsm.getMutations().computeIfAbsent(entry.getKey(),
```

Comment:
At this point, you don't even need to assign it to a variable... can just call `addAll` directly on the result.